src/os/ui/areas.js

```diff
@@ -171,6 +186,40 @@ os.ui.AreasCtrl.prototype.export = function() {
 };
 
 
+/**
+ * Opens the area export menu.
+ *
+ * @export
+ */
+os.ui.AreasCtrl.prototype.openExportMenu = function() {
+  var target = this.element.find('.js-export-group');
+  var menu = os.ui.menu.areaExport.MENU;
+
+  var menuOptions = menu.getRoot()['children'];
```

Comment:
You should use `getChildren()` here rather than referencing `['children']`. This is generally true for all of our typed JS classes. Bracket notation should mostly be used for properties on Angular scopes/controllers that need to be referenced in HTML.

Reply:
There isn't a function that exists right now on `os.ui.menu.Menu` for `getChildren()`, should I make one/